sdk/storage/storage-queue/samples/typescript/iterators.ts

```diff
@@ -23,21 +23,82 @@ async function main() {
   console.log(`List queues`);
 
   // List queues
-  let iter1 = queueServiceClient.listQueues();
   let i = 1;
+  let marker: string | undefined = undefined;
+
+  let iter1 = queueServiceClient.listQueues();
   for await (const item of iter1) {
-    console.log(`Queue${i}: ${item.name}`);
+    console.log(`Queue${i}: ${item!.name}`);
```

Comment:
>! [](start = 34, length = 1)

Why do we need `!` marker here before `name`?

Reply:
TS compiler complains `Object is possibly 'undefined'`.

Reply:
If am correct, `name` should be a required item for QueueItem

Reply:
yeah let's fix this somehow, needing the bangs for this code is problematic I think.